src/AltinnCore/Common/Helpers/Guard.cs

```diff
@@ -53,14 +53,14 @@ public static void AssertArgumentInRange<T>(IList<T> list, int index, string arg
         }
 
         /// <summary>
-        /// The assert org and service arguments are present (not null or whitespace).
+        /// Assert org and app arguments are present (not null or whitespace).
```

Comment:
missing some words to make the sentence complete.. E.g. asserts that arguments are preent..

Reply:
Sure, could add "that" in front of "org".